### PR TITLE
Upgrade to jstransform (from react-tools)

### PR DESCRIPTION
**Pull request by appsforartists**
@petehunt - please merge, version bump, and publish to npm.  :smiley: 

cc @zpao

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "transparently require() jsx from node",
   "main": "index.js",
   "dependencies": {
-    "react-tools": "^0.13.2"
+    "jstransform": ">=11.0.0"
```
Comment (zpao) [inline package.json]: I'm always hesitant to use `>=` in versions. This led to a big mess with `reactify` when we changed the JSX transform. We could change a default, ship a new major version, and then one person's build is vastly different from another's.

Comment (appsforartists) [inline package.json]: I did this for consistency with [jsx-loader](https://github.com/petehunt/jsx-loader/blob/8c9e7115a00ab3a449ea9c50130194f0a3e2ddfe/package.json#L7).

If jstransform is following semver, we could make this "11" and let `npm` worry about getting the latest patch version.  We could also make this a `peerDependency` and let the application developer choose which version of `jstransform` to use.

One frustrating thing about life before `>=` is that new visitors were gated on @petehunt taking the time to version bump.  It'd be nice if there was an easy way to make `jsx-loader` and `node-jsx` use a newer version of `jstransform`.

Should I change this to `"11"` and trust jstransform to bump the minor version for new visitors?

Comment (zpao) [inline package.json]: jstransform is sort of following semver, though perhaps a bit conservatively by bumping version numbers more aggressively (it's tricky with transforms). Honestly I wouldn't expect us to be shipping many more versions - we're in the process of slowly deprecating it.

> were gated on @petehunt

Hey @petehunt, give other people commit access and publishing rights. Like me and @appsforartists.

Comment (appsforartists) [inline package.json]: Commit access is only useful if we have NPM publish rights too.
:simple_smile:

**Comment by petehunt** [comment]
this is backwards compatible, right?

**Comment by appsforartists** [comment]
:clap: 

Thanks @petehunt, @zpao, and @kaustubh-karkare!

**Pull request merged**